pkg/cli/cli.go

```diff
@@ -458,7 +464,15 @@ func cvdCommands(opts *subCommandOpts) []*cobra.Command {
 		},
 	}
 	list.Flags().StringVar(&listFlags.Host, hostFlag, "", "Specifies the host")
-	return []*cobra.Command{create, list}
+	// Pull command
+	pull := &cobra.Command{
+		Use:   "pull [HOST]",
```

Comment:
Does this mean that it can pull everything from a host, but not a single device when a host has several?

Reply:
yes, it would be based on the output of ` cvd host_bugreport` sub-command which is not device specific for now.